### DISCUSSION TITLE
immich on Synology DS220+ Where are my pics stored?

### DISCUSSION THREAD

**Discussion by ReallyRandy** [immich-app/immich#11489] [Q&A] [answered]
In my .env file I have the default "./library" set just for testing. I uploaded 2 picures via the immich browser interface and I uploaded one picure to the "/volume1/docker/immich-app/library" folder. The 2 pics show in the browser interface but not in the docker folder. And the image I uploaded to the Docker folder does **not** show in the browser interface. So where are my 2 photos I uploaded to the browser interface? And why doesn't the "MacWorld" pic show in the interface?
![ScreenShot 2024-07-31 at 9 50 31 AM](https://github.com/user-attachments/assets/3c4003b8-03f0-4aa7-9bd1-b68b47ed0d50)

**Comment by schuhbacca** [comment, marked as answer]
1. "Where are my 2 photos". They are relative to the "./library". So probably wherever you have your env/compose file you will find another folder called library.
2. Why isn't "MacWorld" uploaded. You can't just drop photos into the immich folder and have them be uploaded. They either have to be uploaded via the app/web/cli/api or an external library must be created and added: https://immich.app/docs/features/libraries#external-libraries
